pkg/platform/authentication/auth.go

```diff
@@ -116,8 +116,8 @@ func (s *Auth) SyncRequired() bool {
 func (s *Auth) Sync() error {
 	defer profile.Measure("auth:Sync", time.Now())
 
-	if s.AvailableAPIToken() != "" {
-		logging.Debug("Authenticating with stored API token")
+	if token := s.AvailableAPIToken(); token != "" {
+		logging.Debug("Authenticating with stored API token: %s..", token[0:2])
```

Comment:
Obviously don't want to leak sensitive info, hence why I'm just logging the first 2 characters which seems harmless enough.

Reply:
Maybe create a function `desensitizeToken()` or something that returns `token[0:2]`? That way you don't have to repeat the magic numbers in more than one place. If we need a different method, then it only needs to be changed in one place.